lib/vagrant-libvirt/action/remove_stale_volume.rb

```diff
@@ -1,50 +0,0 @@
-require 'log4r'
```

Comment:
can you explain why you're removing this file? It is used elsewhere in the code see https://github.com/vagrant-libvirt/vagrant-libvirt/blob/master/lib/vagrant-libvirt/action.rb#L344

Reply:
I see you've combined the previous PR in with this one, can you separate them because I'd like to look closer at the error you got in the previous. Then once switching to local variables and I'll merge this.

Reply:
i somehow messed up the PR! Sorry about this!